sympy/functions/elementary/exponential.py

```diff
@@ -277,9 +278,9 @@ def eval(cls, arg):
 
             coeffs, log_term = [coeff], None
             for term in Mul.make_args(terms):
-                if isinstance(term, log):
+                if isinstance(logcombine(term), log):
                     if log_term is None:
-                        log_term = term.args[0]
+                        log_term = logcombine(term).args[0]
```

Comment:
You are calling `logcombine` twice which is inefficient.

Reply:
okk i got you.